src/migrations/20210302170502-add-resources-to-activity-report.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn('ActivityReports',
```

Comment:
Does it make any sense to wrap these in a `Promise.all(...)`? I remember a comment about this from a while ago